cmd/emailreport/README.md

```diff
@@ -10,7 +10,30 @@ file can be specified by command line or it is installed in
 WARNING: The password for the SMTP authentication will be in TOML
 configuration text plane file.
 
-# Usage
+## Options
+`emailreport` takes a number of options.
+
+### Add comments/suggestions in the message body
+```
+-c    comments/suggestions
+```
+
+### Specify TOML configuration file by command line
```

Comment:
Missing 'the':
Specify the TOML configuration file in the command line

Reply:
Ops! you are right :)